uiowa/uids
# 4.x - Banner

Related to https://github.com/uiowa/uids/issues/721. 

## How to test

- Code review
```
nvm use && yarn install && yarn storybook
```
- Review banner controls and confirm that they are all functional and control the property they are meant to control.
- Review banner variants (stories). Review prop/control defaults for each story and provide feedback if they don't make sense.
- Test banner control combinations to ensure options work together.

### Specific things to check:
- A new stories file was added for Media. Confirm that it is not displayed in the Storybook navigation.
- When there is only one button, the link text is rendered as a pseudo button and the banner title is linked.
    - Click to view the full page view of the banner and confirm that entire banner is clickable with the `click-container`. 
 - Confirm when there is more than one button that the links are directly applied to the buttons and not to the banner title.
